### ISSUE TITLE
set:::pictures' size

### ISSUE THREAD

**Issue by Bonsen** [weiliu89/caffe#362]
My pic size is 640*360.... After I get .lmdb, can I just adjust this for 640*360 ? :
________________________________________________
examples/ssd/ssd_pascal.py 
85 # Specify the batch sampler.
86 resize_width = 640                                                                                                   
87 resize_height = 360

297 # parameters for generating priors.
298 # minimum dimension of input image
299 min_dim = 640
____________________________________________

**Comment by weiliu89** [comment]
You should refer to ssd_pascal_orig.py on how to train a model by keeping the image aspect ratio.

Do you expect your loss to go to 0? If so, it means the model is seriously overfitted, which is usually not a good thing. The loss doesn't go too low because there are many data augmentation during training to fight against overfitting.

**Comment by Bonsen** [comment]
@weiliu89 OK, I will see to it, thanks.

**Comment by Bonsen** [comment]
@weiliu89 
I change 
_____
Specify the batch sampler.                                                
resize_width = 300
resize_height = 300
_____
to
Specify the batch sampler.                                                
resize_width = 640
resize_height = 360
_____
in ssd_pascal.py directly....
If the resize_width must be the same as  resize_height ???
